tools/project_manager/diff.php

```diff
@@ -280,8 +289,14 @@ function can_see_names_for_page($projectid, $image)
             }
             $fields .= $round->user_column_name;
         }
-        $query = "SELECT $fields from $projectid WHERE image = '$image'";
-        $res = mysqli_query(DPDatabase::get_connection(), $query) or die(DPDatabase::log_error());
+
+        validate_projectID($projectid);
+        $query = sprintf("
+            SELECT $fields from %s WHERE image = '%s'",
```

Comment:
Insert `$projectid` directly (and let's also capitalize `FROM` while we're here).